lisp/mastodon-auth.el

```diff
@@ -43,6 +43,9 @@
 (defvar mastodon-auth--token-alist nil
   "Alist of User access tokens keyed by instance url.")
 
+(defvar mastodon-auth--acct-alist nil
+  "Alist of acct names keyed by instance url.")
```

Comment:
Is it just the account names or the whole alist of account details?

Reply:
Just the account names